cvat-sdk/cvat_sdk/core/client.py

```diff
@@ -39,21 +43,30 @@ class Config:
 
 class Client:
     """
-    Manages session and configuration.
+    Manages session and configuration, provides resource location.
```

Comment:
What is "resource location"? I don't think it's explained in the docs.

Reply:
Probably, it's something I'd like to see in future. Changed the wording